Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -1666,9 +1666,20 @@ script:
     - description: Comma-separated list of endpoint IDs. Can be retrieved by running the core-get-endpoints command.
       name: endpoint_ids
       required: true
-    - description: Comma-separated list of shell commands to execute.
+    - description: Comma-separated list of shell commands to execute. Use is_raw_command=false to prevent splitting by commas.
       name: commands
       required: true
+    - description: Determines if shell commands are passed in their raw form or split by commas.
+      name: is_raw_command
+      type: Boolean
+      required: false
+    - description: 'Type of shell command. Possible values: "powershell", "null".' 
```

Comment:
`null` isn't the type of the command. Consider `native` or `''` (empty string) instead